js/ui/grid_core/ui.grid_core.adaptivity.js

```diff
@@ -1040,8 +1040,11 @@ module.exports = {
                 _handleTabKeyOnMasterDetailCell: function(eventTarget, direction) {
                     var result = this.callBase(eventTarget, direction);
```

Comment:
Here can define `var $currentCell = this._getFocusedCell();` and add a check with "$currentCell" in "if" operator. 

```
var result = this.callBase(eventTarget, direction),
     $currentCell = this._getFocusedCell();

if(!result && $currentCell) {
```